modules/tinymce/CHANGELOG.md

```diff
@@ -34,7 +34,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `tinymce.settings` global property is no longer set upon initialization #TINY-7359
 - Add-ons such as plugins and themes are no longer constructed using the `new` operator #TINY-8256
 - A number of APIs that were not proper classes, are no longer constructed using the `new` operator #TINY-8322
-- The `mceAddEditor` command can now take an object as its value to specify the id and editor options #TINY-8138
+- The `mceAddEditor` and `mceToggleEditor` command now takes an object as its value to specify the id and editor options #TINY-8138
```

Comment:
```suggestion
- The `mceAddEditor` and `mceToggleEditor` commands now take an object as their value to specify the id and editor options #TINY-8138
```
nitpick, but ?

Reply:
That sounds a lot better so will update it to use your working. Thanks